repository_test.go

```diff
@@ -12,8 +12,8 @@ import (
 )
 
 func init() {
-	t := now().Truncate(time.Second)
-	now = func() time.Time {
+	t := time.Now().Truncate(time.Second)
```

Comment:
this should fix the coverage I think
```suggestion
	t := Now()
```